tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/PathHeader.txt

```diff
@@ -417,6 +417,7 @@
 //supplementalData/metaZones/metazoneInfo/timezone[@type="%A"]/usesMetazone/_mzone			; Supplemental ; Metazone ; $1 ; -∞..∞ ; HIDE
 
 //supplementalData/metaZones/mapTimezones[@type="%A"]/mapZone[@other="%A"][@territory="%A"]/_type			; Supplemental ; Metazone ; $1 ; $2-$3 ; HIDE
+//supplementalData/metaZones/shortIds/shortId[@name="%A"]_alias    ; Supplemental ; Metazone ; $1 ; $2-$3 ; HIDE
```

Comment:
@macchiati What is the test expecting here?

>     TestCompletenessNonLdmlDtd {
>      Error: (TestPathHeader.java:1318) //supplementalData/metaZones/shortIds/shortId[@name="%A"]/_deprecated: Unknown path header[], Supplemental      Unknown metaZones       shortIds/shortId[@name="acre"]/_deprecated
>        No match in PathHeader.txt for //supplementalData/metaZones/shortIds/shortId[@name="acre"]/_deprecated
>        You get only one message for all paths matching //supplementalData/metaZones/shortIds/shortId[@name="%A"]/_deprecated
>        For example, check to see if the field in PathHeader.txt is in PathHeader.PageId.
>        If not, either correct PathHeader.txt or add it to PageId
>        If you have a value attribute, you will need extra _ characters. The value attribute will show at the end with prefixed _, eg [...]/_skeleton.If there can be a value for the path then that element will add _.

Reply:
Every kind of path needs a match in Pathheader.txt. Easiest way to do this is to look for a path like the one you have.

EG //supplementalData/metaZones/mapTimezones[@type="%A"]/mapZone[@other="%A"][@territory="%A"]/_type			; Supplemental ; Metazone ; $1 ; $2-$3 ; HIDE

WARNING: the longest matches have to come first in the file, and you need to have the various combinations of #IMPLIED attributes that appear. That's why you see:

//supplementalData/metaZones/metazoneInfo/timezone[@type="%A"]/usesMetazone[@from="%A"][@to="%A"]/_mzone			; Supplemental ; Metazone ; $1 ; $2-$3 ; HIDE
//supplementalData/metaZones/metazoneInfo/timezone[@type="%A"]/usesMetazone[@from="%A"]/_mzone			; Supplemental ; Metazone ; $1 ; $2..∞ ; HIDE
//supplementalData/metaZones/metazoneInfo/timezone[@type="%A"]/usesMetazone[@to="%A"]/_mzone			; Supplemental ; Metazone ; $1 ; -∞..$2 ; HIDE
//supplementalData/metaZones/metazoneInfo/timezone[@type="%A"]/usesMetazone/_mzone			; Supplemental ; Metazone ; $1 ; -∞..∞ ; HIDE

For the attributes for shortId, only preferred may or might not occur. So two patterns should suffice.

Reply:
You have $2-$3, bu there is only one variable in the regex (%A), so only a $1. 

Best to look at the bcp47 equivalents for the pattern.

Reply:
The test results show what the paths turn into. The value parameters turn into separate items
//supplementalData/metaZones/metazoneIds/metazoneId[@shortId="%A"]/_deprecated
//supplementalData/metaZones/metazoneIds/metazoneId[@shortId="%A"]/_longId

Reply:
So you need to provide values for these. The values for all three can be done with the following, I think.

//supplementalData/metaZones/shortIds/shortId[@name="%A"]_%E    ; Supplemental ; Metazone ; $1 ; $2 ; HIDE